openpype/settings/entities/schemas/projects_schema/schemas/schema_global_publish.json

```diff
@@ -203,6 +203,69 @@
                                             "default": 0,
                                             "minimum": 0,
                                             "maximum": 100000
+                                        },
+                                        {
+                                            "key": "letter_box",
+                                            "label": "Letter box",
+                                            "type": "dict",
+                                            "checkbox_key": "enabled",
+                                            "children": [
+                                                {
+                                                    "type": "boolean",
+                                                    "key": "enabled",
+                                                    "label": "Enabled",
+                                                    "default": false
+                                                },
+                                                {
+                                                    "key": "ratio",
+                                                    "label": "Letter box ratio",
```

Comment:
Maybe this should just be `ratio` since its under "Letterbox" and the ratio applies to the pillar mode as well.

Reply:
that's true actually.